boskos/cmd/cli/cli.go

```diff
@@ -141,17 +149,17 @@ Examples:
 			}
 			resource, err := acquireFunc(options.acquire.requestedType, options.acquire.requestedState, options.acquire.targetState)
 			if err != nil {
-				fmt.Fprintf(cmd.ErrOrStderr(), "failed to acquire a resource: %v\n", err)
+				logrus.Errorf("failed to acquire a resource: %v", err)
```

Comment:
In all of these cases this is not expected to use `logrus` -- users on a CLI want to have normal print output, not JSON-formatted structured logs

Reply:
Done.